nano/node/nodeconfig.cpp

```diff
@@ -352,6 +354,10 @@ nano::error nano::node_config::deserialize_json (bool & upgraded_a, nano::jsonco
 		json.get<unsigned long> ("vote_generator_delay", delay_l);
 		vote_generator_delay = std::chrono::milliseconds (delay_l);
 
+		unsigned threshold_l = vote_generator_threshold;
+		json.get<unsigned> ("vote_generator_threshold", threshold_l);
+		vote_generator_threshold = threshold_l;
```

Comment:
These 3 lines can be simplified to `json.get<unsigned> ("vote_generator_threshold", vote_generator_threshold);` as there's no conversion to time units or anything like that - it'll still use the default.

Reply:
Thank you!